src/planwise/client/db.cljs

```diff
@@ -39,8 +39,10 @@
                  :filters {:type #{}
                            :ownership #{}
                            :services #{}}
-                 :count 16
-                 :total 125}}
+                 :count 0
+                 :total 4944
+                 :facilities []} ;; TODO the path [:facilities :facilities] is confusing
```

Comment:
On projects we have `[:projects :list]` to avoid this problem, but a better name is also welcome